script.js

```diff
@@ -1,4 +1,72 @@
+
+// Project 1: scissors paper stone 
+// user input scissors or paper, or stone
+// input validation for "scissors" "paper" "stone"
+// assign 0,1,2 to scissors, paper, stone
+// random generate number 0,1,2
+// check input vs comPlay
+// winCondition: scissors > paper || paper > stone || stone > scissors 
+// drawCondition: input == computerRandom
+// updated for formatting 
```

Comment:
When submitting code it is good practice to remove comments from the code unless they directly explain what the line of code does.